xen/drivers/passthrough/iommu.c

```diff
@@ -129,7 +129,7 @@ static void __init parse_iommu_param(char *s)
     } while ( ss );
 }
 
-int iommu_domain_init(struct domain *d)
+int iommu_domain_init(struct domain *d, bool_t use_iommu)
```

Comment:
Well, it is a nasty "feature" of the github. I do review this stuff by patches, but github does not allow to comment a particular patch from the series.

For me the patch ["iommu: Pass additional use_iommu argument to iommu_domain_init()"](https://github.com/xen-troops/xen/pull/48/commits/6a6efe072110542ab5f394af0af428e8a737ddaf) looks really strange: you add an unused parameter to the function. Could it be a subject to squash with some further patch?